From 5ba43b6e9b2e85bfe3fa44fa176bce16c30097fb Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 20 Mar 2023 09:34:23 +0000
Subject: [PATCH] stress-af-alg: remove check for HAVE_LIB_IPSEC_MB
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This was erroneously added in an earlier commit. Remove it.

Issue: https://github.com/ColinIanKing/stress-ng/issues/274
Fixes: 6c1191b29ac83 ("stress-af-alg: check if HAVE_LIB_IPSEC_MB is defined")

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
---
 stress-af-alg.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/stress-af-alg.c b/stress-af-alg.c
index 7fb2c53a..a8d01e4f 100644
--- a/stress-af-alg.c
+++ b/stress-af-alg.c
@@ -48,7 +48,6 @@ static const stress_opt_set_func_t opt_set_funcs[] = {
 
 #if defined(HAVE_LINUX_IF_ALG_H) &&	\
     defined(HAVE_LINUX_SOCKET_H) &&	\
-    defined(HAVE_LIB_IPSEC_MB) &&	\
     defined(AF_ALG)
 
 #if !defined(SOL_ALG)
-- 
2.30.2

